Modules/Loadable/Markups/qSlicerMarkupsModuleWidget.cxx

```diff
@@ -2257,14 +2257,9 @@ void qSlicerMarkupsModuleWidget::onRightClickActiveMarkupTableWidget(QPoint pos)
   menu.addAction(resetFiducialAction);
   QObject::connect(resetFiducialAction, SIGNAL(triggered()),
       this, SLOT(onResetMarkupPushButtonClicked()));
-  QAction* unsetFiducialAction =
-      new QAction(QString("Unset position of highlighted fiducial(s)"), &menu);
-  menu.addAction(unsetFiducialAction);
-  QObject::connect(unsetFiducialAction, SIGNAL(triggered()),
-      this, SLOT(onUnsetMarkupPushButtonClicked()));
 
   QAction* missingFiducialAction =
-      new QAction(QString("Set position missing for highlighted fiducial(s)"), &menu);
+      new QAction(QString("Skip position for highlighted fiducial(s)"), &menu);
```

Comment:
"Skip position" is not very clear. Maybe "mark as missing"?

Reply:
"Mark as position is missing" isn't very clear in how that is different from a cleared position. "Skip" defines what functionally is going to happen with this control point in placement mode, however just stating "Missing" doesn't indicate how it will be used any differently from "Clear".

Reply:
Using a predefined set of named landmarks (template) and explicitly marking some of them as not available is a feature that I saw in several of our projects. The need is not related to minor implementation details such as indexing, but it is about differentiating between "position is undefined" and "position is defined to be unknown". You may of want to skip points for various reasons (not necessarily because it is missing), so "skip" may be more accurate than "missing". I don't have a problem with using the "skip" word. What I find confusing is that you don't skip the position, but you skip _definition_ of the position, or _placement_ of the position, or exclude the point from placement.

Reply:
I can image that deleting a point can be acceptable in certain workflows. However, this approach has two important shortcomings:
1. If you delete the point then there is no information on why it was removed.
2. Removing a point is an irreversible operation. When landmarking result is reviewed, point positions may be modified, which includes changing, setting, unsetting positions. It is possible to implement restoring of skipped landmarks in each landmarking module, but since this need is such a common need, it makes sense to implement it only once, cleanly, in Slicer core.

Reply:
So you may set it to be skipped, but then later decide that it shouldn't be skipped?

Reply:
Trying to think of clear wording for the action when dealing with the control point lists of line nodes and angle nodes.

Reply:
I've updated to "Skip placement..."
![image](https://user-images.githubusercontent.com/15837524/135538948-63104e00-798b-4262-93e1-95583486bfdc.png)

Reply:
I like "Skip placement".

I also like the icons for Cut/Copy/Paste actions. What do you think about adding the state icons there so that the actions can be recognized more easily?

Reply:
Good idea! Done.
![image](https://user-images.githubusercontent.com/15837524/135560220-af12dcae-f9f4-4a9f-8e96-438d536958bc.png)